flytekit/annotated/container_task.py

```diff
@@ -63,7 +63,7 @@ def execute(self, **kwargs) -> Any:
         return None
 
     def get_container(self, settings: RegistrationSettings) -> _task_model.Container:
-        env = settings.env
+        env = {**settings.env, **self.environment} if self.environment else settings.env
```

Comment:
this works?  cool, new to me.  and the second one takes precedence?

Reply:
this adds them together. is that undesirable?

Reply:
that is desirable, just wondering what happens if the same key is in both.